packages/twenty-server/src/engine/core-modules/workspace-invitation/services/workspace-invitation.service.ts

```diff
@@ -234,7 +235,7 @@ export class WorkspaceInvitationService {
           link: link.toString(),
           workspace: { name: workspace.displayName, logo: workspace.logo },
           sender: { email: sender.email, firstName: sender.firstName },
-          serverUrl: this.environmentService.get('SERVER_URL'),
+          serverUrl: ApiUrl.get(),
```

Comment:
shouldn't this be a front url computed based on:
if Multidomain: workspaceDomainName + domain
if not: FRONT_BASE_URL